synapse/lib/stormtypes.py

```diff
@@ -7452,6 +7474,12 @@ async def move(self, viewiden):
         tdef['view'] = viewiden
         tdef['user'] = useriden
 
+        try:
+            s_trigger.reqValidTdef(tdef)
+            await self.runt.snap.core.reqValidStorm(tdef['storm'])
+        except (s_exc.SchemaViolation, s_exc.BadSyntax) as exc:
```

Comment:
Can we capture the `mesg` or `str(exc)` here so that the user facing exception is accessable?
We also need to either terminate the exception chain or raise the exception with a chain, otherwise this exception looks like it was the result of handling those two exception cases.

`raise fooErr() from None` or `raise fooErr from exc`